ci/test_narwhals.sh

```diff
@@ -20,7 +20,7 @@ rapids-logger "pytest narwhals"
 NARWHALS_VERSION=$(python -c "import narwhals; print(narwhals.__version__)")
 git clone https://github.com/narwhals-dev/narwhals.git --depth=1 -b "v${NARWHALS_VERSION}" narwhals
 pushd narwhals || exit 1
-rapids-pip-retry install -U -e . pytest-env "hypothesis>=6.131.7"
+rapids-pip-retry install -U -e . pytest-env "hypothesis>=6.131.7" "nvtx>=0.2.1,<0.2.12",
```

Comment:
Please move these dependencies to a new dependency list in `dependencies.yaml`. Call it `test_python_narwhals` and add it to the existing `test_python_narwhals` file key. https://github.com/rapidsai/cudf/blob/8c66eac1de0b2a988bc5136aa875596e35b0fe0f/dependencies.yaml#L389

Reply:
Done in https://github.com/rapidsai/cudf/pull/18971/commits/f3c2dbde72068a32ee63e8b6a9ebd2475bf02c59